applications/adaptive-learning/students/ssd-mobilenet-v2-student.Dockerfile

```diff
@@ -0,0 +1,31 @@
+#This container will install TensorFlow Object Detection API and its dependencies in the /model/research/object_detection directory
+
+FROM tensorflow/tensorflow:1.15.0-gpu-py3
```

Comment:
as this code is placed in neuralet repo, I suggest mentioning the supported hardware platform (e.g. amd64 with Nvidia GPU) in the README.